app/models/user.py

```diff
@@ -26,12 +27,15 @@ def validate_email(cls, value):
 
 # Properties to receive on item creation
 class UserCreate(UserBase):
-    pass
+    password: str | None = Field(default=None)
```

Comment:
Sugiero ponerle la misma regla que le ponés en `password.py`, para que no rompa la tabla al insertar password cuando le ponés por ejemplo un password vacío.

```suggestion
    password: str | None = Field(min_length=1, max_length=255)
```

Reply:
Se realiza la sugerencia

Reply:
Se agrego un test para cuando se da contraseña vacia

Reply:
Voy a dejar pasar el hecho de que un password vacío en principio no debería llegar desde el front al endpoint de esa manera, y en caso que suceda, Debiéramos finalizar el endpoint, sin agregar el usuario y devolviendo un 422, no un 201.

No voy a volver a poner en wip por este tema, así que agrego una tarea al backlog al respecto.

Se crea la [SCRUM-370](https://tpii-padel-pals.atlassian.net/browse/SCRUM-370) para tal fin.

Reply:
Ahora no falla el endpoint cuando le paso un password vacío, pero no inserta el password vacío y el usuario queda sin contraseña en la tabla.
Esto se va a solucionar cuando se haga igualmente la 370. Dejo así por ahora.